common/scala/src/main/resources/application.conf

```diff
@@ -169,6 +169,9 @@ whisk {
         # Select from one of the supported
         # https://azure.github.io/azure-cosmosdb-java/1.0.0/com/microsoft/azure/cosmosdb/ConsistencyLevel.html
         consistency-level = "Session"
+
+        # TTL duration. By default no TTL is set unless explicitly configured
+    #   time-to-live      = 60 s
```

Comment:
It may not be obvious that this example (60s) means documents will only reside in any of these collections for 60s. 
I would 
* emphasize this a bit more (e.g. ENABLING THIS VALUE  MEANS YOUR DATA WILL NOT BE PERMANENTLY STORED)
* disable this for WhiskAuth + WhiskEntity (is there any good use case for these?)
* or maybe only include any sample config only for WhiskActivation collection (this would be a more typical real config right?)

Reply:
For `WhiskAuth` and `WhiskEntity` yes it does not make sense to modify the default. added checks now to ensure that its only used for `WhiskActivation`